string motor_power_action

---
bool is_success
